metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py

```diff
@@ -17,9 +17,9 @@
 assert AIRFLOW_PATCHED
 
 if TYPE_CHECKING:
-    from airflow import DAG
     from airflow.models import DagRun, TaskInstance
 
+    from airflow import DAG
```

Comment:
why'd this change?

Reply:
This changes happens after running lint fix command:
./gradlew :metadata-ingestion-modules:airflow-plugin:lintFix
and Github CL is getting failed due to this changes.
Can you tell me why same command is behaving different on local and github cl?